civis/ml/_model.py

```diff
@@ -1042,17 +1140,11 @@ def from_existing(cls, train_job_id, train_run_id='latest', client=None):
 
         # Set prediction template corresponding to training template
         template_id = int(container['from_template_id'])
-        p_id = _PRED_TEMPLATES.get(template_id)
-        if p_id is None:
-            warnings.warn('Model %s was trained with a newer version of '
-                          'CivisML than is available in the API client '
-                          'version %s. Please update your API client version. '
-                          'Attempting to use an older version of the '
-                          'prediction code. Prediction will either fail '
-                          'immediately or succeed.'
-                          % (train_job_id, __version__), RuntimeWarning)
-            p_id = max([v for k, v in _PRED_TEMPLATES.items()
-                        if k not in REGISTRATION_TEMPLATES])
```

Comment:
Removing this warning since this situation should be impossible to arise, given that we get the registration template ID that corresponds to the given training template ID.